docs/book/src/topics/crd-changes/v1alpha7-to-v1alpha8.md

```diff
@@ -176,4 +176,51 @@ In v1alpha8, this will be automatically converted to:
 
 `Subnets` allows specifications of maximum two `SubnetFilter` one being IPv4 and the other IPv6. Both subnets must be on the same network. Any filtered subnets will be added to `OpenStackCluster.Status.Network.Subnets`.
 
-When subnets are not specified on `OpenStackCluster` and only the network is, the network is used to identify the subnets to use. If more than two subnets exist in the network, the user must specify which ones to use by defining the `OpenStackCluster.Spec.Subnets` field.
\ No newline at end of file
+When subnets are not specified on `OpenStackCluster` and only the network is, the network is used to identify the subnets to use. If more than two subnets exist in the network, the user must specify which ones to use by defining the `OpenStackCluster.Spec.Subnets` field.
+
+#### ⚠️ Change to managedSecurityGroups
+
+The field `managedSecurityGroups` is now a `SecurityGroupsSpec` object rather than a boolean.
+The `SecurityGroupsSpec` object has a `enabled` boolean to enable/disable the managed security groups feature.
+
+Also, we can now add security group rules that authorize traffic from all nodes via `allNodesSecurityGroupRules`.
+It takes a list of security groups rules that should be applied to selected nodes.
+The following rule fields are mutually exclusive: `remoteManagedGroups`, `remoteGroupID` and `remoteIPPrefix`.
+
+Valid values for `remoteManagedGroups` are `controlplane`, `worker` and `bastion`.
+
+```yaml
+managedSecurityGroups: true
+```
+
+becomes
+
+```yaml
+managedSecurityGroups:
+  enabled: true
+```
+
+To apply a security group rule that will allow BGP between the control plane and workers, you can follow this example:
+
+```yaml
+managedSecurityGroups:
+  enabled: true
+  allNodesSecurityGroupRules:
+  - remoteManagedGroups:
+    - controlplane
+    - worker
+    direction: ingress
+    ethertype: IPv4
+    portRangeMin: 179
+    portRangeMax: 179
+    protocol: tcp
+    description: "Allow BGP between control plane and workers"
+  ```
+
+#### ⚠️ Calico CNI
+
+Historically we used to create the necessary security group rules for Calico CNI to work. This is no longer the case.
+Now the user needs to create the security group rules manually by using the `managedSecurityGroups.allNodesSecurityGroupRules` feature.
```

Comment:
This gave me an idea that the user needed to pre-create those rules, so I propose the following instead:

```suggestion
Now the user needs to request creation of the security group rules by using the `managedSecurityGroups.allNodesSecurityGroupRules` feature.
```